ghcide/src/Development/IDE/Main.hs

```diff
@@ -161,13 +161,14 @@ commandP plugins =
              (fullDesc <> progDesc "Print config supported by the server with default values")
 
     pluginCommands = mconcat
-        [ command (T.unpack pId) (Custom "." <$> p)
+        [ command (T.unpack pId) (Custom <$> p)
         | (PluginId pId, PluginDescriptor{pluginCli = Just p}) <- ipMap plugins
         ]
 
 
 data Arguments = Arguments
-    { argsOTMemoryProfiling     :: Bool
+    { argsProjectRoot           :: IO FilePath
```

Comment:
I prefer the first version with the `Maybe FilePath` type here. The second version breaks `lsp --cwd` since `argsProjectRoot` is not invoked in the `LSP` command.

Reply:
Lol... good point. I preferred the first way, but since we should always reach getHieDbLoc with a FilePath I tried to alleviate the problem. I'm good with reverting back.